interfaces/evse_security.yaml

```diff
@@ -103,6 +103,16 @@ cmds:
       ocsp_response:
         description: OCSPResponse class as defined in IETF RFC 6960. DER and then base64 encoded
         type: string
+  retrieve_ocsp_cache:
+    description: Command to retrieve the OCSP cache file path of the specified certificate hash
+    arguments:
+      certificate_hash_data:
+        description: Certificate hash data that identifies the certificate for which the cache should be retrieved
+        type: object
+        $ref: /evse_security#/CertificateHashData
+    result:
+      description: Path of data related to the certificate hash, if present
```

Comment:
Like the changed response type for the CSR generation, I would also prefer a dedicated response type here that carries more information (status + ocsp result (if present)).

Reply:
I am not entirely sure what error could be present here, we can only have one state for the OCSP cache related to a certificate. Present or NotPresent. A std::optional seems to be taking into account those 2 cases. Is there anything else that I might be forgetting when handling this case?

Reply:
The issue is that the function returns an `std::string` and not and `std::optional<std::string>` . For this reason we need a more complex return type like for the CSR generation

Reply:
Understood, makes sense, since I've forgot that the API does not allow std::optional return types.